.github/workflows/tests-cleanup.yml

```
@@ -0,0 +1,77 @@
+name: TestsCleanUp
+on:
+  push:
+    branches:      
+      - tf-*      
+  schedule:
+    - cron: '0 0 * * *' # Runs dayly at 12:00:00
+  workflow_dispatch:
+jobs:
+  build:    
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v1    
+    - name: Run pws script to delete SQS transport acceptance tests AWS artifacts
+      env: 
+        LIMITED_DELETE_RIGHTS_ACCOUNT_KEY: ${{ secrets.TMP_CHRISTIAN_AWS_ACCESS_KEY_ID }}
+        LIMITED_DELETE_RIGHTS_ACCOUNT_SECRET_KEY: ${{ secrets.TMP_CHRISTIAN_AWS_SECRET_KEY_ID }}
+      shell: pwsh
+      run: |
+        $limitedDeleteRightsAccessKey = $env:LIMITED_DELETE_RIGHTS_ACCOUNT_KEY
+        $limitedDeleteRightsSecretKey = $env:LIMITED_DELETE_RIGHTS_ACCOUNT_SECRET_KEY
+
+        if (!(Get-Module AWSPowerShell.NetCore)) { 
+            Write-Host "Installing AWSPowerShell.NetCore..." 
+            Install-Module -name AWSPowerShell.NetCore -Scope CurrentUser -Force -AllowClobber
+        }
+
+        Write-Host "Importing module..."
+        Import-Module AWSPowerShell.NetCore -Force
+
+        Write-Host "The following version of AWSPowerShell.NetCore is being used"
+        Get-AWSPowerShellVersion
+
+        #$dateFilter = ([System.DateTime]::UtcNow).AddHours(-24)
+        $dateFilter = ([System.DateTime]::UtcNow).AddMinutes(-1)
```

Another "before merging"
```suggestion
        $dateFilter = ([System.DateTime]::UtcNow).AddHours(-24)
```